### DISCUSSION TITLE
size of videos "wrong" under Safari at loading

### DISCUSSION THREAD

**Discussion by godbout** [tailwindlabs/tailwindcss#8539] [Help]
see video below:

https://user-images.githubusercontent.com/121373/172478800-b48e95d6-3504-452c-8e79-95f2e1f9e569.mp4

this issue happens only in Safari. Brave, Chrome and Firefox work fine.

i was able to pinpoint this to those lines in the base css generated by Tailwind (3.0.24):

```
/*
1. Make replaced elements `display: block` by default. (https://github.com/mozdevs/cssremedy/issues/14)
2. Add `vertical-align: middle` to align replaced elements more sensibly by default. (https://github.com/jensimmons/cssremedy/issues/14#issuecomment-634934210)
   This can trigger a poorly considered lint error in some tools but is included by design.
*/

img,
svg,
video,
canvas,
audio,
iframe,
embed,
object {
  display: block; /* 1 */
  vertical-align: middle; /* 2 */
}

/*
Constrain images and videos to the parent width and preserve their intrinsic aspect ratio. (https://github.com/mozdevs/cssremedy/issues/14)
*/

img,
video {
  max-width: 100%;
  height: auto;
}
```

removing those lines solve the issue.

i've tried two things:
1. override those settings with default browser values. but i couldn't make it work.
2. force an aspect ratio on the videos and images. the issue is 1) that screws up the designs (margins, paddings etc.) 2) the rounding doesn't work anymore on the videos.

any idea how it's possible to solve this, or what i'm doing wrong? thanks.

**Comment by Gregory210** [comment]
I will endeavor to figure it out for extra.

[Flare Account
](https://www.aceflareaccount.net/)

**Comment by godbout** [comment]
i somehow "solved" the issue by setting a non lazy loaded poster to lazy loaded videos. it's slower to load that i would like, but at least it's not jittery.

i don't think the issue has anything to do with the lazy loading itself, and it's failing on a simple page with a normally loaded video. so not sure my explanation is totally accurate. but after spending 6 hours on this, i'll take it.
